3rdparty/musl/CMakeLists.txt

```diff
@@ -12,21 +12,21 @@ endif()
 
 if (WIN32)
```

Comment:
The whole logic to find Bash doesn't need to be repeated. We can just put it in `3rdparty/CMakeLists.txt` at the top.

Reply:
Reason for duplication is because the bash invocation in libunwind will need to be removed. But will move this to the top.

Reply:
Oh, that's right, that makes sense. You can leave it as-is with a TODO.

Reply:
Moved it to the top. Seems cleaner in any case.